sanket/javascript/day17/assignment3/sanketA3_Q5.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    no1: <input type="text" id="no1" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no1").value);
+        let f = 1;
+        for (i = 1; i <= n; i++) f = f * i;
+        document.getElementById("p").innerHTML = `Factorial  = ${f} `;
```

Comment:
remove extra space

Reply:
ma'am thank you for you reply i'll remove extra space asap

Reply:
extra space is present before =or after Factorial